Resources/Prototypes/Entities/Structures/Wallmounts/switch.yml

```diff
@@ -7,26 +7,26 @@
     snap:
     - Wallmount
   components:
-  - type: WallMount
-    arc: 360
-  - type: Clickable
-  - type: InteractionOutline
-  - type: Physics
-  - type: Sprite
-    sprite: Structures/Wallmounts/switch.rsi
-    state: on
-  - type: SignalSwitch
-  - type: UseDelay
-    delay: 0.5 # prevent light-toggling auto-clickers.
-  - type: Rotatable
-  - type: Construction
-    graph: SignalSwitchGraph
-    node: SignalSwitchNode
-  - type: Fixtures
-  - type: SignalTransmitter
-    outputs:
-      On: []
-      Off: []
+    - type: WallMount
+      arc: 360
+    - type: Clickable
+    - type: InteractionOutline
+    - type: Physics
+    - type: Sprite
+      sprite: Structures/Wallmounts/switch.rsi
+      state: on
+    - type: SignalSwitch
+    - type: UseDelay
+      delay: 0.5 # prevent light-toggling auto-clickers.
+    - type: Rotatable
+    - type: Construction
+      graph: SignalSwitchGraph
+      node: SignalSwitchNode
+    - type: Fixtures
+    - type: SignalTransmitter
+      outputs:
+        On: []
+        Off: []
```

Comment:
Would you be able to restore these lines? It looks like all that was changed was the indentation.

Reply:
I did that for consistent formatting. I just wanted to tidy things up a bit while I was already editing the file.

Reply:
Actually, I took a look around and it seems the standard here is no indent. So, I'll revert that and remove the indent from the ones that were inconsistent.